readthedocs/templates/builds/build_detail.html

```diff
@@ -161,18 +161,20 @@
           </p>
         </div>
       {% endif %}
-      {% if build.finished and not build.using_latest_config %}
+
+      {# This message is not dynamic and only appears when loading the page after the build has finished #}
+      {% if build.finished and build.deprecated_config_used %}
         <div class="build-ideas">
           <p>
             {% blocktrans trimmed with config_file_link="https://docs.readthedocs.io/page/config-file/v2.html" %}
-              <strong>Configure your documentation builds!</strong>
-              Adding a <a href="{{ config_file_link }}">.readthedocs.yaml</a> file to your project
-              is the recommended way to configure your documentation builds.
-              You can declare dependencies, set up submodules, and many other great features.
+              <strong>Your builds will stop working soon!</strong><br/>
```

Comment:
I like this heading. This is the exact thing I'd love to have in our new notification pattern.

A general note, but `<br>` should almost never be used. It complicates styling as it isn't semantic. I'd instead split up text using two `<div>` elements to provide separation.